services/models/istio_validation.go

```diff
@@ -1,51 +1,42 @@
 package models
 
-// IstioTypeValidations represents a set of IstioNameValidations grouped per Istio ObjectType.
-// It is possible that different object types have same name, but names per ObjectType are unique.
-type IstioTypeValidations map[string]*IstioNameValidations
+// IstioValidationKey is the key value composed of an Istio ObjectType and Name.
```

Comment:
Looks like a good change to me, cleaner model.

However I suspect there are some missing parts: the validations object is queried in handler/services' function `ServiceIstioValidations`, which serializes the returned `map[string]*IstioValidation` (now `map[IstioValidationKey]*IstioValidation`) as json. I wonder how `IstioValidationKey` is going to be translated as a json map key. And kiali-ui should be adapted as well, I guess.